taxonomies/ingredients.txt

```diff
@@ -9954,7 +10106,7 @@ fr:lait de bufflonne pasteurisé
 
 <en:milk
 en:evaporated milk
-de:Kondensmilch
+de:evaporierte Milch, evaporierte Kondensmilch
```

Comment:
Is `de:Kondensmilch` now deleted?

Reply:
No, it was there twice, once as ```en:evaporated milk``` and once as ```en:condensed milk```.